integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy

```diff
@@ -149,81 +151,109 @@ class AuthorizationServerIT extends ITSupport {
         assertNotPresent(client.listAuthorizationServers(), createdAuthorizationServer)
     }
 
-    // Policy operations
-
     @Test
-    void listAuthorizationServerPoliciesTest() {
+    void deactivateAuthorizationServerTest() {
         String name = "java-sdk-it-" + UUID.randomUUID().toString()
 
-        AuthorizationServerPolicy authorizationServerPolicy = client.instantiate(AuthorizationServerPolicy)
-            .setType(PolicyType.OAUTH_AUTHORIZATION_POLICY)
-            .setName("Test Policy")
-            .setDescription("Test Policy")
-            .setPriority(1)
-            .setConditions(client.instantiate(PolicyRuleConditions)
-                .setClients(client.instantiate(ClientPolicyCondition)
-                    .setInclude([OAuth2Scope.MetadataPublishEnum.ALL_CLIENTS.name()])))
-
         AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
             client.instantiate(AuthorizationServer)
                 .setName(name)
                 .setDescription("Test Authorization Server")
-                .setAudiences(["api://example"]))
+                .setAudiences(["api://example"])
+        )
         registerForCleanup(createdAuthorizationServer)
-
         assertThat(createdAuthorizationServer, notNullValue())
+        assertThat(createdAuthorizationServer.getStatus(), equalTo(AuthorizationServer.StatusEnum.ACTIVE))
 
-        AuthorizationServerPolicy createdAuthorizationServerPolicy = createdAuthorizationServer.createPolicy(authorizationServerPolicy)
+        createdAuthorizationServer.deactivate()
 
-        AuthorizationServerPolicyRule authorizationServerPolicyRule = createdAuthorizationServerPolicy.createPolicyRule(createdAuthorizationServer.getId(),
-            client.instantiate(AuthorizationServerPolicyRule)
+        // delete operation may not effect immediately in the backend
+        sleep(getTestOperationDelay())
+
+        AuthorizationServer retrievedAuthorizationServer = client.getAuthorizationServer(createdAuthorizationServer.getId())
+        assertThat(retrievedAuthorizationServer.getId(), equalTo(createdAuthorizationServer.getId()))
```

Comment:
Also assert retrievedAuthorizationServer for not null.

Reply:
done